test/fixtures/mock-sdk/lib/toolkit/loader.js

```diff
@@ -759,14 +769,14 @@ const Loader = iced(function Loader(options) {
     load: { enumerable: false, value: options.load || load },
     // Main (entry point) module, it can be set only once, since loader
     // instance can have only one main module.
-    main: new function() {
+    main: function() {
```

Comment:
I'm not too worried about changing the semantics of this code, since it's just fixture code used by the tests. But yeah we could ignore this one if you think so.